app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -71,4 +75,36 @@ def stale_encrypted_email?
     return false unless email.present?
     @_encrypted_email.stale?
   end
+
+  def phone
+    return unless encrypted_phone.present?
+    @_encrypted_phone ||= EncryptedAttribute.new(
```

Comment:
thoughts on putting L 81-85 in a separate / private method? I feel like this method and `email` both have a lot going on, might clarify to break out some smaller methods